x-pack/plugins/cloud/server/config.ts

```diff
@@ -22,6 +22,10 @@ const configSchema = schema.object({
   apm: schema.maybe(apmConfigSchema),
   base_url: schema.maybe(schema.string()),
   cname: schema.maybe(schema.string()),
+  // Cloud service provider
+  csp: schema.maybe(
```

Comment:
Q: do we need to expose it to the browser? Or this is only needed on the server side for now?

Reply:
yes, it's about the browser side